automatic/zotero/tools/chocolateyUninstall.ps1

```diff
@@ -2,7 +2,7 @@
 
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
-  softwareName  = 'Zotero'
+  softwareName  = "$env:ChocolateyPackageName*"
```

Comment:
Again, can you confirm that this matches the name that shows up when you manually uninstall the application?